### PR TITLE
Added locale en-GB

### PR DESCRIPTION
**Pull request by glintik**
Hi,

Locale en-GB is added and tested.

### CODE REVIEW

File: src/locale/en-GB/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import formatRelative from '../en-US/_lib/formatRelative/index.js'
+import localize from '../en-US/_lib/localize/index.js'
+import match from '../en-US/_lib/match/index.js'
+
+/**
+ * @type {Locale}
+ * @category Locales
+ * @summary English locale (United Kingdom).
+ * @language English
+ * @iso-639-2 eng
```
Comment (leshakoss) [inline src/locale/en-GB/index.js]: Please add @author tag, like this:

```
 * @author Alex [@glintik]{@link https://github.com/glintik}
```

File: src/locale/en-GB/_lib/formatLong/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import buildFormatLongFn from '../../../_lib/buildFormatLongFn/index.js'
+
+var formatLong = buildFormatLongFn({
+  LT: 'HH:mm',
+  LTS: 'HH:mm:ss',
+  L: 'DD/MM/YYYY',
+  LL : 'D MMMM YYYY',
```
Comment (leshakoss) [inline src/locale/en-GB/_lib/formatLong/index.js]: Please remove extra space after key here and in two following lines. https://travis-ci.org/date-fns/date-fns/jobs/276343759#L490-L492

**Comment by glintik** [comment]
Sorry, forgot to run eslint. Fixed.

**Comment by leshakoss** [comment]
@glintik while we're waiting for @kossnocorp to release this as a new version, could you please add `@author` tag to jsdoc like this: https://github.com/date-fns/date-fns/pull/563#discussion_r139303185

**Comment by kossnocorp** [comment]
It's released as `v2.0.0-alpha.5`: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha5.

Thank you!

**Pull request merged**